packages/headless/src/utils/highlight.ts

```diff
@@ -72,3 +89,17 @@ export function highlightString(params: HighlightParams): string {
   }
   return highlighted;
 }
+
+/**
+ * Cast the parameter to an array of HighlightKeywords to be used in the 'highlightString' function.
+ * @param highlightKeywords The highlights keywords as an array of string. This should be was is returned by the API
+ */
+export function highlightKeywordString(
+  highlightKeywords: string[]
+): HighlightKeyword[] {
+  return highlightKeywords.map(
+    (keyword): HighlightKeyword => {
+      return (keyword as unknown) as HighlightKeyword;
```

Comment:
This does not look right, a `HighlightKeyword` should have an offset and length?